### PR TITLE
Feature/sc 18779/fix banner on clark to work with secured

### PR DESCRIPTION
**Pull request by JosueMolinaMorales**
This PR completes [Fix Banner on CLARK to work with SecurEd Downtime lambda](https://app.shortcut.com/clarkcan/story/18779/fix-banner-on-clark-to-work-with-secured-downtime-lambda) by implementing the new secured downtime service allowing a downtime banner to be added without a deployment of the entire client.

Demo where interval for hitting the service is 10 seconds:

https://user-images.githubusercontent.com/72173743/222810225-0d32c066-36e7-4923-8f07-7ddd1653c9a3.mov

### CODE REVIEW

File: src/app/admin/admin.component.ts [hunk 1]
```diff
@@ -38,11 +38,11 @@ export class AdminComponent implements OnInit, OnDestroy {
     private messagesService: MessagesService
   ) {}
 
-  ngOnInit() {
+  async ngOnInit() {
     // hide CLARK navbar
     this.navbarService.hide();
 
-    if (this.messagesService.getStatus()) {
+    if (!!(await this.messagesService.getDowntime()).message) {
```
Comment (Maliyaa) [inline src/app/admin/admin.component.ts:45]: Is this why the banner says lorem ipsum right now, as seen in the demo video?

Comment (JosueMolinaMorales) [reply src/app/admin/admin.component.ts:45]: If i understand your question right, no this helps display the banner in the admin dashboard. The code that enables the banner is in `app.component.ts`

**Review by bereketsega — APPROVED** [review]

File: src/app/admin/admin.component.ts (hunk 1 above)
Comment (Maliyaa) [reply src/app/admin/admin.component.ts:45]: Ooo right.

**Review by Maliyaa — APPROVED** [review]

**Review by pzalep1 — APPROVED** [review]
Looks good!

**Review by mfranz1 — APPROVED** [review]

**Pull request merged**